### ISSUE TITLE
How do I print the validation loss for each epoch?

### ISSUE THREAD

**Issue by AndreaPi** [open-mmlab/mmdetection#6131]
As per subject: with a default config I'm able to print train loss, bbox mAP, seg mAP, etc. (I'm doing instance segmentation), but I'd like to visualize the val loss too in Tensorboard. How can I log the val loss too? I searched the docs with "print validation loss", but I had no luck.

**Comment by ZwwWayne** [comment]
See the example here. https://mmdetection.readthedocs.io/en/latest/tutorials/customize_runtime.html?highlight=workflow#customize-workflow
